Marlin/Conditionals.h

```diff
@@ -46,6 +46,21 @@
 
   #define CONFIGURATION_LCD
 
+  #if ENABLED(CartesioUI)
+    #define DOGLCD
+    #define SDSUPPORT
+    #define ULTIPANEL
+    #define NEWPANEL
+    #define DEFAULT_LCD_CONTRAST               90
+    #define ENCODER_PULSES_PER_STEP            2
+    #define ENCODER_STEPS_PER_MENU_ITEM        1
+    #define SPEAKER
+    #define LCD_FEEDBACK_FREQUENCY_HZ          1000  // this is the tone frequency the buzzer plays when on UI feedback. ie Screen Click
+    #define LCD_FEEDBACK_FREQUENCY_DURATION_MS 100 // the duration the buzzer plays the UI feedback sound. ie Screen Click. 0 to disable buzzer feedback 
```

Comment:
With upcoming changes to buzzer handling, you may not need to set these two options. You might want to set `LCD_USE_I2C_BUZZER` instead, if it applies. Otherwise the default for a speaker is 5000Hz for 2ms (a very quick "blip" sound).